app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -59,4 +57,32 @@ def raw_safe_delete_volume
 
     EmsRefresh.queue_refresh(ext_management_system)
   end
+
+  def self.params_for_create(provider)
+    services = provider.storage_services.map { |service| {:value => service.ems_ref, :label => service.name} }
+    @params_for_create ||= {
```

Comment:
Since the `services` are dynamic we shouldn't cache the result (aka don't do `@params_for_create ||=`).  This leads to stale dropdown options if someone e.g. creates or deletes one of the storage_services while the application is running.